solutions/dashboard/config/logstash/LogstashCronjob.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: dashboard-logstash
+  namespace: dashboard
+spec:
+  schedule: "0 */2 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: dashboard-logstash
+              image: docker.io/sanket02092001/customlog:1.0.0
```

Comment:
can we add this image to our local registry and use it?

Reply:
Yes, we are planning to do that. 
For now, we are creating this image by adding another layer of MongoDB driver files to the existing logstash image. 
But, later we will modify the Dockerfile to support dynamically adding the driver files (using wget) into the image while building.

Reply:
Okay